src/brokers/utils/add-broker.ts

```diff
@@ -168,7 +212,9 @@ type ArtemisReducerActions =
   | SetNamespaceAction
   | SetReplicasNumberAction
   | UpdateAcceptorFactoryClassAction
-  | UpdateConnectorFactoryClassAction;
+  | UpdateConnectorFactoryClassAction
+  | UpdateEditorStateAction
+  | SetModelAction;
```

Comment:
The values are alphabetically ordered, can you place this new `SetModelAction` before `SetReplicasNumberAction` please?

Reply:
I think you mean 'before setNamespaceAction` as M is before N. :)